tests/pthread/test_pthread_tls.cpp

```diff
@@ -0,0 +1,18 @@
+#include <thread>
+#include <cstdio>
+
+thread_local int tls;
```

Comment:
Do thread local structs work as well? (in C or C++)

Reply:
Yes, it's being tested now.